src/fsharp/fsi/fsiAnyCPU.exe.config

```diff
@@ -12,6 +12,15 @@
           oldVersion="2.0.0.0-4.4.1.0"
           newVersion="4.4.1.0"/>
       </dependentAssembly>
+      <dependentAssembly>
```

Comment:
@KevinRansom  We are referencing version 4.0.0.0, so fsc shouldn't be affected by this change. BUT this is needed for NGEN. Do you have concerns for adding this redirection here?